### PR TITLE
Docs(web-twig): List Deprecations in the web twig package #DS-1148

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 1]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
 
 > Please follow these steps to safely upgrade your design system to Spirit Design System v3 components.
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:5]: As mention in the `web` PR, I think only package version should be here. The common version is rather a marketing thing and I'm hesitating to legalize it by using it in our code…

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 2]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
...
+
+The `id` prop will be mandatory for the `Dropdown` component.
+
+#### Migration Guide
+
+Add `id` prop to the `Dropdown` component.
+
+### Tooltip and Dropdown Placements
+
+The `Tooltip` and `Dropdown` components no longer support non-flows relative placements.
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:37]:
```suggestion
The `Tooltip` and `Dropdown` components no longer support non-flow-relative placements.
```

Isn't the adjective _flow-relative_…? I don't know 😄

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 3]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
...
+- `<Dropdown placement="top-left" … />` → `<Dropdown placement="top-start" … />`
+- `<Dropdown placement="top-right" … />` → `<Dropdown placement="top-end" … />`
+- `<Dropdown placement="right-top" … />` → `<Dropdown placement="right-start" … />`
+- `<Dropdown placement="right-bottom" … />` → `<Dropdown placement="right-end" … />`
+- `<Dropdown placement="bottom-left" … />` → `<Dropdown placement="bottom-start" … />`
+- `<Dropdown placement="bottom-right" … />` → `<Dropdown placement="bottom-end" … />`
+- `<Dropdown placement="left-top" … />` → `<Dropdown placement="left-start" … />`
+- `<Dropdown placement="left-bottom" … />` → `<Dropdown placement="left-end" … />`
+
+### Tooltip composition
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:61]: Either Title Case or Sentence case should be used for headings. A good tip for our MD linter.

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 4]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
...
+
+The `TooltipWrapper` component name will be replaced by `Tooltip`. Use the `TooltipWrapper`
+component now and rename it to `Tooltip` in the next major version. Component `Tooltip` is
+deprecated and is replaced by `TooltipPopover`.
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:65]: How about something like:

> The `Tooltip` component structure will be changed, so `Tooltip` (former the optional `TooltipWrapper`) becomes the wrapping component. Similarly, `Tooltip` (the tip itself) has been renamed to `TooltipPopover`.

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 5]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
...
+  <Tooltip id="my-tooltip">
+    Hello there!
+  </Tooltip>
+</TooltipWrapper>
+```
+
+Use:
+
+```twig
+<TooltipWrapper>
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:87]: Not a `Tooltip`…? 🤔

File: packages/web-twig/DEPRECATIONS-v3.md [hunk 6]
```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
...
+  </a>
+  <TooltipPopover id="my-tooltip">
+    Hello there!
+  </TooltipPopover>
+</TooltipWrapper>
+```
+
+See [`Tooltip` documentation][tooltip-readme] for more details and examples.
+
+### Breadcrumbs goBackTitle prop
```
Comment (adamkudrna) [inline packages/web-twig/DEPRECATIONS-v3.md:104]:
```suggestion
### Breadcrumbs `goBackTitle` prop
```

File: packages/web-twig/DEPRECATIONS-v3.md (hunk 3 above)
Comment (literat) [reply packages/web-twig/DEPRECATIONS-v3.md:61]: I think that `Title Case` will be required: https://github.com/lmc-eu/code-quality-tools/blob/main/.textlintrc.js#L4

**Review by literat — APPROVED** [review]
After incorporating all the comments from @adamkudrna, it will be looking good 👍

File: packages/web-twig/DEPRECATIONS-v3.md (hunk 1 above)
Comment (crishpeen) [reply packages/web-twig/DEPRECATIONS-v3.md:5]: it is already there. We used this versioning in our migration guides in the summer.

Comment (crishpeen) [reply packages/web-twig/DEPRECATIONS-v3.md:5]: I will leave it like it is. Feel free to open a topic on our Thursday meetings

**Pull request merged**